docs/usage/getting-started/private-packages.md

```diff
@@ -88,6 +88,60 @@ Here is an example of some host rules:
 Renovate applies theses `hostRules` to every HTTP(s) request which is sent, so they are largely independent of any platform or datasource logic.
 With `hostRules` in place, private package lookups should all work.
 
+### GitHub (and Enterprise) repo scoped credentials
+
+If you need to use seperate credentials for a single GitHub repo which is different from the default, then you can configure `hostRules` like this:
+
+```json
+{
+  "hostRules": [
+    {
+      "matchHost": "https://api.github.com/repos/org/repo",
+      "token": "abc123"
+    },
+    {
+      "matchHost": "https://gihub.domain.com/api/v3/repos/org/repo",
+      "token": "abc123"
+    }
+  ]
+}
+```
+
+Renovate will use those credentials for all requests to `org/repo`.
+
+Here's an example for `gomod` with private github.com repos.
```

Comment:
```suggestion
#### Example for gomod

Here's an example for `gomod` with private github.com repos.
```

How about putting a new heading here? It helps separate the general explanation and the specific example.